src/test/java/com/github/freeclimbapi/TranscribeUtteranceTest.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * FreeClimb API
+ * FreeClimb is a cloud-based application programming interface (API) that puts the power of the Vail platform in your hands. FreeClimb simplifies the process of creating applications that can use a full range of telephony features without requiring specialized or on-site telephony equipment. Using the FreeClimb REST API to write applications is easy! You have the option to use the language of your choice or hit the API directly. Your application can execute a command by issuing a RESTful request to the FreeClimb API. The base URL to send HTTP requests to the FreeClimb REST API is: /apiserver. FreeClimb authenticates and processes your request.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: support@freeclimb.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.github.freeclimbapi;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.threeten.bp.OffsetDateTime;
+import org.threeten.bp.LocalDate;
+
+import java.util.*;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.io.File;
+import java.math.BigDecimal;
+import java.net.URI;
+import java.net.URISyntaxException;
+import com.github.freeclimbapi.enums.*;
+import com.github.freeclimbapi.models.*;
+
+/**
+ * Model tests for TranscribeUtterance
+ */
+public class TranscribeUtteranceTest {
+  private final TranscribeUtterance model = new TranscribeUtterance();
+
+  /**
+   * Test the property 'actionUrl'
+   */
+  @Test
+  public void actionUrlTest() {
+    model.setActionUrl("TEST_STRING");
+    Assert.assertEquals("TEST_STRING", model.getActionUrl());
+
+  }
+
+  /**
+   * Test the property 'playBeep'
+   */
+  @Test
+  public void playBeepTest() {
+    model.setPlayBeep(false);
+    Assert.assertEquals(false, model.getPlayBeep());
+
+  }
+
+  /**
+   * Test the property 'record'
+   */
+  @Test
+  public void recordTest() {
+
```

Comment:
Noticed the test for this is missing. Wondering if we need a boolean flag here.

Reply:
We will, I've added the records test to each of the SDKs manually, for a future improvement, will note this down.

In next commit, I'll add the record test in manually.